packages/featuregates/bundle/config/values.yaml

```diff
@@ -2,9 +2,8 @@
 ---
 namespace: tkg-system
 deployment:
-  hostNetwork: true
-  nodeSelector:
-    node-role.kubernetes.io/control-plane: ""
```

Comment:
Description only mentions `hostNetwork` change. Why are we removing nodeSelector too?

Reply:
that was the initial workaround fix that @12345lcr added with this pr https://github.com/vmware-tanzu/tanzu-framework/pull/3983, so reverting that